.github/workflows/reusable-markdown-link-check.yml

```diff
@@ -12,15 +12,14 @@ jobs:
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
-      - name: Install markdown-link-check
-        # https://github.com/tcort/markdown-link-check/issues/297
-        run: npm install -g markdown-link-check@3.11.2
-
-      - name: Run markdown-link-check
-        run: |
-          find . -type f \
-                 -name '*.md' \
-                 -not -path './CHANGELOG.md' \
-                 -not -path './licenses/*' \
-                 -not -path '*/build/*' \
-               | xargs .github/scripts/markdown-link-check-with-retry.sh
+      - uses: lycheeverse/lychee-action@v2
+        with:
+          # remove version after next release of lychee-action
+          lycheeVersion: latest
+          # excluding links to pull requests and issues is done for performance
+          args: >
+            --include-fragments
```

Comment:
this option doesn't exist - but it's done

Reply:
specific args are documented here: https://github.com/lycheeverse/lychee?tab=readme-ov-file#commandline-parameters